db/migrations/20190315134628_add_bandwidth_limit_column_to_displaygroup_migration.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * Copyright (C) 2018 Xibo Signage Ltd
+ *
+ * Xibo - Digital Signage - http://www.xibo.org.uk
+ *
+ * This file is part of Xibo.
+ *
+ * Xibo is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * any later version.
+ *
+ * Xibo is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Xibo.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+use Phinx\Migration\AbstractMigration;
+
+class AddBandwidthLimitColumnToDisplaygroupMigration extends AbstractMigration
+{
+    /** @inheritdoc */
+    public function change()
+    {
+        $displayGroupTable = $this->table('displaygroup');
+
+        if (!$displayGroupTable->hasColumn('bandwidthLimit')) {
+            $displayGroupTable
+                ->addColumn('bandwidthLimit', 'integer', ['default' => 0, 'null' => true])
```

Comment:
are we sure that we need the field nullable if the default value is 0? if so, do we handle a mixture of nulls and 0's in this field when we query for it?